specification/electronic-prescription-service-api.yaml

```diff
@@ -274,6 +276,334 @@ paths:
                   value:
                     $ref: components/examples/SendErrorInvalidDateFormatResponse.yaml
           headers: {}
+  /Convert:
+    post:
+      operationId: convert-prescription
+      summary: Convert a prescription to a format used for signing
+      description: Convert a prescription from FHIR format into a HL7 V3 representation which can be used to verify signing operations.
+      tags:
+        - prescribing
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: components/schemas/Bundle.yaml
+            examples:
+              example:
+                value:
+                  $ref: components/examples/ConvertSuccessRequest.json
+        description: ''
+      responses:
+        '200':
+          description: |
+            Successfully converted prescription.
+
+            ### Body
+
+            application/xml
+
+            **Example**
+
+                <ParentPrescription classCode="INFO" moodCode="EVN">
```

Comment:
Discussed putting the response body in the description instead of in the responseBody section:
- It would be a lot of work to add a schema for the XML.
- This endpoint is a temporary one until we have a dedicated endpoint for suppliers to use to verify their signature implementations.
- We're going to be updating the response to be FHIR compliant anyway, e.g. by returning a Binary or Parameters Resource.